lib/t_web/channels/feed_channel.ex

```diff
@@ -14,7 +13,7 @@ defmodule TWeb.FeedChannel do
       user_id = String.downcase(user_id)
       %{screen_width: screen_width} = socket.assigns
 
-      gender_preferences = Accounts.list_gender_preferences(user_id)
+      feed_filter = Feeds.get_feed_filter(user_id)
```

Comment:
how is it kept up-to-date with profile?

Reply:
feed channel can subscribe for profile updates and refetch feed filter in `handle_info` when the even arrives

Reply:
I just rejoin the feedChannel on client: https://github.com/getsince/test5/blob/rails-fork/Since/Feed/FeedCoordinator.swift#L207
but you're right, probably this approach is cleaner